ocaml/idl/datamodel.ml

```diff
@@ -562,6 +562,19 @@ let _ =
   error Api_errors.user_is_not_local_superuser [ "msg" ]
     ~doc:"Only the local superuser can execute this operation" ();
 
+  (* SR-IOV errors *)
+  error Api_errors.network_sriov_already_enabled ["PIF"]
+    ~doc:"You tried to enable network SR-IOV on PIF which is already enabled" ();
```

Comment:
I would just write `"The PIF selected for the SR-IOV network is already enabled."`